packages/binding-http/src/http-server.ts

```diff
@@ -216,6 +216,16 @@ export default class HttpServer implements ProtocolServer {
             let propertyNamePattern = this.updateInteractionNameWithUriVariablePattern(propertyName, thing.properties[propertyName].uriVariables);
             let href = base + "/" + this.PROPERTY_DIR + "/" + propertyNamePattern;
             let form = new TD.Form(href, type);
+            
+            if(tdTemplate && tdTemplate.properties[propertyName] && tdTemplate.properties[propertyName].forms) {
+              for (let formTemplate of tdTemplate.properties[propertyName].forms) {
+                // TODO match HTTP only?
```

Comment:
As I said in my previous https://github.com/eclipse/thingweb.node-wot/pull/187#issuecomment-598839011 it might worth exploring the possibility to move this logic outside of this class.

Reply:
This is a very good point. I will extract the code in some sort of a helper class. The PR is still work in progress and I hope I get to work on it the next days! Will keep you informed!